docs/editing.md

```diff
@@ -126,6 +128,45 @@ Kind regards,
 JOSS Editor.
 ```
 
+### Query scope of submission
+
+```
+:wave: thanks for your submission to JOSS. From a quick inspection of this submission it's not entirely obvious that it meets our [submission criteria](https://joss.readthedocs.io/en/latest/submitting.html#submission-requirements). In particular, this item:
+
+> - Your software should have an obvious research application
```

Comment:
Perhaps leave this line a placeholder so editor could fill in with any of the requirements instead of just this particular one?